src/Objects/Scene.cpp

```diff
@@ -56,21 +94,38 @@ void Scene::RemoveGameObject(GameObject &gameObject) {
 }
 
 Scene::Scene(Camera &&mainCamera) {
-    AddCamera(std::move(mainCamera));
+    addCamera(std::move(mainCamera));
 }
 
 Scene::Scene() {
-    AddCamera(Camera());
+    addCamera(Camera());
 
 }
 
-std::vector<std::unique_ptr<Camera>> &Scene::GetCameras() {
+std::vector<std::unique_ptr<Camera>> &Scene::getCameras() {
     return cameras;
 }
 
-void Scene::AddCamera(Camera &&camera) {
+void Scene::addCamera(Camera &&camera) {
     cameras.push_back(std::make_unique<Camera>(camera));
 }
 
+std::string Scene::getSignature() {
+    return signature;
+}
+
+std::string Scene::generateSignature() {
```

Comment:
Is het hierdoor niet mogelijk dat dezelfde signiture ontstaat? Kans is klein maar wel mogelijk toch?